### ISSUE TITLE
The loss is sometimes over 1e17

### ISSUE THREAD

**Issue by 20171130** [gcorso/torsional-diffusion#2]
Hi, I am trying to train a torsional diffusion model. Since torus.score_norm()  is very small when sigma is large, the training loss can be enormous. Is this the expected behavior? Do you suggest using a smaller sigma_max for numerical stability?

**Comment by 20171130** [comment]
I noticed that this can be resolved by scaling the score with the score_norm.
